.github/workflows/build-and-push-docker.yml

```diff
@@ -0,0 +1,166 @@
+name: 'Build & Push Selected Docker Images to Docker Hub'
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  DOCKER_REPO_BROKER_SERVICE: orochinetwork/zkdb-broker-service
+  DOCKER_REPO_PROOF_SERVICE: orochinetwork/zkdb-proof-service
+  DOCKER_REPO_SERVERLESS: orochinetwork/zkdb-serverless
```

Comment:
lack of `NPM_TOKEN: ${{ secrets.NPM_ACCESS_TOKEN }}`

Reply:
Okay I will change the step to assign npm token to this section, Currently i'm putting it at `build` step